backup/backup.go

```diff
@@ -6,13 +6,13 @@ import (
 	"fmt"
 	"time"
 
+	_ "github.com/gnolang/gno/gno.land/pkg/sdk/vm"
```

Comment:
Maybe let's add a comment to explain it.

Reply:
done.